src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,203 @@
+            preparedStatement.setLong(1, driverId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                Long carId = resultSet.getLong("car_id");
+                Car car = get(carId)
+                        .orElseThrow(() -> new DataProcessingException("Couldn't find car"
+                                + " with such id: " + carId));
+                cars.add(car);
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get all cars by driver:"
+                    + driverDao.get(driverId));
+        }
+    }
+
+    private void deleteDrivers(Car car) {
+        String deleteRequest = "DELETE FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(deleteRequest)) {
+            statement.setLong(1, car.getId());
+            statement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete all drivers which use this car - "
+                    + car, e);
+        }
+    }
+
+    private void insertDrivers(Car car) {
+        String query = "INSERT INTO cars_drivers (car_id, driver_id) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, car.getId());
+            for (Driver driver : car.getDrivers()) {
+                preparedStatement.setLong(2, driver.getId());
+                preparedStatement.executeUpdate();
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't add drivers: " + car.getDrivers(), e);
+        }
+    }
+
+    private void setDriversList(Car car) {
+        String query = "SELECT driver_id FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1, car.getId());
+            ResultSet resultSet = preparedStatement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                Long driverId = resultSet.getLong("driver_id");
+                Driver driver = driverDao.get(driverId)
+                        .orElseThrow(() -> new DataProcessingException("No driver with such id: "
+                        + driverId));
+                drivers.add(driver);
+            }
+            car.setDrivers(drivers);
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get drivers for car: " + car, e);
+        }
```

Comment:
the same here, you don't need to call n-times `get` of `DriverDao`, write only 1 query.